ivc/src/ivc/columns.rs

```diff
@@ -336,80 +341,86 @@ impl ColumnIndexer for IVCColumn {
 
     fn to_column(self) -> Column {
         match self {
+            IVCColumn::FoldIteration => Column::Relation(0),
             IVCColumn::BlockSel(i) => {
                 assert!(i < N_BLOCKS);
                 Column::FixedSelector(i)
             }
 
             IVCColumn::Block1Input(i) => {
                 assert!(i < 2 * N_LIMBS_SMALL);
-                Column::Relation(i)
+                Column::Relation(i).add_rel_offset(1)
```

Comment:
In the IVC PR, a lot of this copy-paste is avoided by applying `add_rel_offset` to the whole block.